### PR TITLE
OnlineStatusFunctionalityProjectPassed

### PR DESCRIPTION
**Pull request by ZehraAkan**

### CODE REVIEW

**Review by rnkaya — COMMENTED** [review]
All good congrats.

**Comment by ecnracr** [comment]
everything looks good.

File: src/test/java/com/meetsky/runners/CukesRunner.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
                 "junit:target/cucumber-report.xml",
                 "rerun:target/rerun.txt"
         },
+
```
Comment (ecnracr) [inline src/test/java/com/meetsky/runners/CukesRunner.java:19]: conflict has resolved

**Review by ecnracr — APPROVED** [review]
everything looks good conflict has been resolved

**Pull request merged**